### ISSUE TITLE
Class diagram might be too wordy

### ISSUE THREAD

**Issue by chiewyx** [chiewyx/pe#8]
![image.png](https://raw.githubusercontent.com/chiewyx/pe/main/files/31a7cbf3-5840-45ff-91c7-aab478d3a9c0.png)

Some unnecessary methods and attributes should be removed for readability

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
All methods are necessary as they are at least called once and the reason for so many methods was to stick to the SLAP hard principle and keep each method short and simple. All attributes are also necessary.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------
